app/controllers/problems_controller.rb

```diff
@@ -46,6 +46,10 @@ def show
     @comment = Comment.new
   end
 
+  def xhr_sparkline
```

Comment:
Question. Should this action go in the "API"? Any opinions on that?

Reply:
i like keeping this out of the API so folks don't confuse it for something that they might call from a script or something. it returns html, and it's built just for this rails ajaxy integration. i'm not recommending this necessarily, but one day, if the app were rewritten in reactjs, many of the existing controller actions would return json instead of html, and we could make this one fit that model as well.